src/core_landice/mode_forward/mpas_li_iceshelf_melt.F

```diff
@@ -1183,7 +1184,7 @@ subroutine li_grounded_face_melt_ismip6(domain, err)
       !-----------------------------------------------------------------
       integer :: iCell, iNeighbor, iEdge, nEmptyNeighbors
       real (kind=RKIND), pointer :: rhoi
-      integer, pointer :: nCellsSolve
+      integer, pointer :: nCells, nCellsSolve
```

Comment:
Should nCellsSolve (and its 'get' routine below) be entirely deleted?  (if it's not used elsewhere)

Reply:
It's used for the loop in which faceMeltSpeed is calculated. I don't know whether it's more appropriate to loop over nCells or nCellsSolve there.